src/bundlers/node/genezioRuntimeHandlerGenerator.ts

```diff
@@ -34,7 +34,7 @@ if (!genezioClass) {
                 },
                 id: 0,
             }),
-            headers: { 'Content-Type': 'application/json', 'X-Powered-By': 'genezio' }
+            headers: { 'Content-Type': 'application/json', 'X-Powered-By': 'genezio','Access-Control-Allow-Origin': '*', 'Access-Control-Allow-Headers': '*', 'Access-Control-Allow-Methods': '*' }
```

Comment:
Why are we adding the header in all the responses? Shouldn't it be present only when the method is OPTIONS?

Reply:
I tried to put it only in OPTIONS but it doesn't work, I still get a CORS error, the only way for this to work is if the responses also have CORS headers present. I think it might have something to do with the fact that when we used the CloudFormationBuilder on Lambda we set the CORS policy for the ApiGateway like this



```
        this.addResource(apiGatewayResourceName, {
            Type: "AWS::ApiGatewayV2::Api",
            Properties: {
                Name: apiGatewayName,
                ProtocolType: "HTTP",
                Description: `API Gateway for Genezio Project ${apiGatewayName}}`,
                CorsConfiguration: {
                    AllowOrigins: ["*"],
                    AllowMethods: ["*"],
                    AllowHeaders: ["*"],
                    MaxAge: 10800,
                },
            },
        });
```

I i think the ApiGateway handled adding CORS to each lambda response.

We don't have the same mechanism for the Runtime so we need to put it manually in each response.